app/src/main/java/com/elavon/converge/config/BusinessUpdatedReceiver.java

```diff
@@ -13,5 +15,6 @@ public void onReceive(Context context, Intent intent) {
         Log.d(TAG, "onReceive: " + intent.getAction());
         Intent serviceIntent = new Intent(context, LoadBusinessIntentService.class);
         context.startService(serviceIntent);
+        ElavonConvergeProcessorApplication.getInstance().loadUser();
```

Comment:
this only happens when business is updated - which doesn't happen that often. Which means what yo have cached as the user might not be accurate if a user switches account.

Reply:
@ppalavilli this also happens when the user switches account. In fact, every time user switches account, poynt.intent.action.BUSINESS_UPDATED gets triggered instead of android.accounts.LOGIN_ACCOUNTS_CHANGED. Which is strange.

Reply:
this sounds wrong - can you file a jira for PoyntServices for this ? it looks like the business_updated will get called when the user login forces business change (for multi-mid use case) but that shouldn't happen for single mid terminals.